docs/references/plugin_lifecycle_markdown.md

```diff
@@ -0,0 +1,88 @@
+Lifecycle of a Plugin
+=====================
+
+Prerequisites
+-------------
+
+This overview refers to Plugins, in reference to any new `PluginV2`
+class developed with the SDK.
+
+Overview
+--------
+
+The lifecycle of a `Plugin` in Alteryx Designer can be broken down into
+2 different types of runs:
+
+1.  Workflow Run
+2.  Update Only Run
+
+Workflow Run
+------------
+
+When you select the **Run** button in Designer, an instance of your
+`PluginV2` class is created for each of your tools on the canvas.
+
+The `PluginV2` is constructed via its `__init__` method, just like in
+normal Python. An instance of a `ProviderV2` class is given to the
+plugin. Any resources required to read from and write to Designer can be
+obtained via methods/properties on the `ProviderV2`.
+
+In Alteryx Designer, input anchors can have 1 or many connections. If
+you have any input connections, in the next stage of the lifecycle
+consists of each of those input connections sends data downstream. From
```

Comment:
TW review:

In Alteryx Designer, input anchors can have 1 or many connections. If
you have any input connections, the next stage of the lifecycle
consists of each of those input connections sending data downstream. From

Reply:
fixed